scripts/repo-link/mechanics/mining.dsc

```diff
@@ -30,7 +30,7 @@ gielinor_mining_handler:
       # $ ████████ [ ADD CHECK FOR MODIFIERS ] ████████
       - define mining_result <util.random.int[0].to[100]>
       - if <[mining_result].add[<[mining_level_bonus]>]> >= success_chance:
-        - give <[ore_type]>
+        - give <proc[item].context[ore_type]>
```

Comment:
`ore_type` is an invalid item - you'd want the definition `<[ore_type]>` defined up at `define ore_type <npc.flag[type]>_ore`
```suggestion
        - give <proc[item].context[<[ore_type]>]>
```